src/main/java/uk/gov/justice/hmpps/prison/api/model/ImageDetail.java

```diff
@@ -33,6 +33,10 @@ public class ImageDetail {
     @NotNull
     private Long imageId;
 
+    @Schema(requiredMode = RequiredMode.REQUIRED, description = "Active", example = "false")
+    @NotNull
+    private Boolean active;
```

Comment:
seem odd to annotate with `NonNull` but yet use the object `Boolean` rather than the primitive.  Also `active` doesn't really mean anything to me here.  What does it mean for an image to be `active`?

Reply:
Thanks again for the feedback, we've created https://github.com/ministryofjustice/prison-api/pull/1601 to address it!